lib/lex/inline.js

```diff
@@ -104,7 +104,47 @@ InlineLexer.prototype.output = function(src) {
       continue;
     }
 
+    // html
+    ///*
+    if (cap = this.rules.html.exec(src)) {
+      // Found a paragraph
+      if(cap[1] === 'p' && cap[2]) {
+        // Opening tag
+        out += cap[0].substring(0, cap[0].indexOf(cap[2]));
+        // In between the tag
+        out += this.output(cap[2]);
+        // Outer tag
+        out += cap[0].substring(cap[0].indexOf(cap[2])+cap[2].length);
+        // Advance parser
+        src = src.substring(cap[0].length);
+        continue;
+      }
+
+      // Found a link
+      if(cap[1] === 'a' && cap[2] && !this.inLink) {
+        console.log('Parsing =', cap[2], 'to', this.output(cap[2]));
```

Comment:
Yes, there's some other debug stuff in there too (stuff in comments rather than deleted, I wanted to make sure this worked for you before deleting that and merging).